eo-maven-plugin/src/main/java/org/eolang/maven/tojos/ForeignTojo.java

```diff
@@ -404,4 +404,22 @@ public boolean equals(final Object other) {
     public int hashCode() {
         return Objects.hash(this.delegate);
     }
+
+    /**
+     * Return the attribute from the tojo.
+     * @param attribute The attribute from ForeignTojos.Attribute.
+     * @return The attribute.
+     */
+    private String attribute(final ForeignTojos.Attribute attribute) {
+        final String path = this.delegate.get(attribute.key());
+        if (path == null) {
+            throw new IllegalStateException(
```

Comment:
@Yanich96 the message we print here to the user won't help them understand what happened, I believe. Maybe we should explain a bit more, suggesting to run the `verify` goal of the plugin first?